src/test/common/steps/address-lookup-steps.js

```diff
@@ -26,3 +32,13 @@ Then(/^I am shown a button to enter my address manually$/, async function () {
   const buttonText = await pages.selectAddress.getSubmitButtonText()
   expect(buttonText).to.be.equal('Enter address manually')
 })
+
+Then(/^I am shown a list of addresses$/, async function () {
+  const addressOptions = await pages.selectAddress.getAddressOptions()
+  expect(addressOptions.length).to.be.equal(11)
```

Comment:
I think it might be worthwhile checking that the addresses returned are relevant, so maybe check that each address shown has the correct postcode?

Reply:
Done.